l1-contracts/src.ts/deploy-test-process.ts

```diff
@@ -89,7 +92,6 @@ export async function initialEraTestnetDeploymentProcess(
   // deploy the verifier first
   await initialBridgehubDeployment(deployer, extraFacets, gasPrice, true, 1);
   await initialBridgehubDeployment(deployer, extraFacets, gasPrice, false, 1);
-  deployer.addresses.Create2Factory = "0x1bba393e38a2CD88638F972D67D73599c094f814"; // this should already be deployed, we need to fix it to fix ERA diamond address
```

Comment:
why have you removed it?

Reply:
I didn't quite understand the purpose, as `deployer.addresses.Create2Factory` already is set to the last Create2 contract deployed.

Reply:
Also, I removed it because it breaks when another contract is deployed in the middle (TestnetVerifier + Verifier) as the address of Create2Factory now changed.